bindings/python/py_src/safetensors/torch.py

```diff
@@ -185,16 +185,19 @@ def load_model(model: torch.nn.Module, filename: Union[str, os.PathLike], strict
         filename (`str`, or `os.PathLike`):
             The filename location to load the file from.
         strict (`bool`, *optional*, defaults to True):
-            Wether to fail if you're missing keys or having unexpected ones
+            Whether to fail if you're missing keys or having unexpected ones.
             When false, the function simply returns missing and unexpected names.
+        device (`Dict[str, any]`, *optional*, defaults to `cpu`):
```

Comment:
```suggestion
        device (`str`, *optional*, defaults to `cpu`):
```

is it dict or str ?

Reply:
I took the exact same definition as in load_file docstring: https://github.com/huggingface/safetensors/blob/b947b59079a6197d7930dfb535818ac4896113e8/bindings/python/py_src/safetensors/torch.py#L291

Reply:
Made the change in https://github.com/huggingface/safetensors/pull/449/commits/d723ff7be806a6a696763b80554579d99791524b :) (and used the `Union[Dict[str, any], str]` type).

Reply:
Go down to the rust code [here](https://github.com/huggingface/safetensors/blob/b947b59079a6197d7930dfb535818ac4896113e8/bindings/python/src/lib.rs#L241):
```rs
#[derive(Debug, Clone, PartialEq, Eq)]
enum Device {
    Cpu,
    Cuda(usize),
    Mps,
    Npu(usize),
    Xpu(usize),
}
```

Reply:
why us it `Union[Dict[str, any], str]` rather that `str` ? 

Doesn't the enum mean that value is str rather than dict? 
```rust
enum Device {
    Cpu,
    Cuda(usize),
    Mps,
    Npu(usize),
    Xpu(usize),
}
```

Reply:
Yes sorry so I went back to the source code and it looks like the expected type should be `Union[str, int]`. Here is the code:
- expecting [string](https://github.com/huggingface/safetensors/blob/b947b59079a6197d7930dfb535818ac4896113e8/bindings/python/src/lib.rs#L251)
- expecting [int](https://github.com/huggingface/safetensors/blob/b947b59079a6197d7930dfb535818ac4896113e8/bindings/python/src/lib.rs#L295) (which has been discussed/added in https://github.com/huggingface/safetensors/pull/35#discussion_r1005666280) => defaults to "Cuda:<int_value>".

I have updated the types accordingly in https://github.com/huggingface/safetensors/pull/449/commits/3bad1e2618c16a5db70a4e02b4fcf1ebb8641a95. Let's wait for @Narsil's return just to be sure.